src/shared/dissect-image.c

```diff
@@ -2287,6 +2287,8 @@ int dissected_image_mount_and_warn(
         assert(m);
 
         r = dissected_image_mount(m, where, uid_shift, uid_range, userns_fd, flags);
+        if (r == -ENOENT)
+                return log_error_errno(r, "A mount point directory in the image is missing.");
```

Comment:
hmm, ENOENT can happen in other cases too. maybe change this to be a bit more vague, i.e.:

```File or directory not found. (Maybe a directory that shall be used as mount point is mising?)```

or so?